src/Core/ServiceWrapper/Main.cs

```diff
@@ -536,7 +536,14 @@ public static void Run(string[] _args, ServiceDescriptor? descriptor = null)
             bool inCliMode = _args.Length > 0;
 
             // If descriptor is not specified, initialize the new one (and load configs from there)
-            descriptor ??= new ServiceDescriptor();
+            if(_args.Length > 1)

```

Comment:
Let me explain how this will behave:

****Without config file specified:****

- The user executes `winsw start`, `inCliMode` evaluates to `true`.
- SCM executes `winsw`, `inCliMode` evaluates to `false`.

****With config file specified:****

- The user executes `winsw start path/to/config`, `inCliMode` evaluates to `true`. `_args[1]` is `start`.
- SCM executes `winsw`, `inCliMode` evaluates to `false`. WinSW doesn't know a config file is specified.